Sources/QuranAudioKit/AudioPlayer/GaplessAudioRequestBuilder.swift

```diff
@@ -68,7 +68,15 @@ final class GaplessAudioRequestBuilder: QuranAudioRequestBuilder {
                     for (offset, verse) in suraTimings.verses.enumerated() {
                         // start from 0 (beginning) if first ayah of the sura
                         let endTime = offset == suraTimings.verses.count - 1 ? suraTimings.endTime : nil
-                        let frame = AudioFrame(startTime: offset == 0 && verse.ayah.ayah == 1 ? 0 : verse.time.seconds, endTime: endTime?.seconds)
+                        
+                        var startTimeSeconds = offset == 0 && verse.ayah.ayah == 1 ? 0 : verse.time.seconds
+                        
+                        // avoiding bismillah when repeated.
+                        if requestRuns > .one, ayahs.count == .zero {
```

Comment:
nit:
1) `requestRuns != .one` seems to do the trick without the need to add Comparable conformance.
2) `ayahs.count == .zero` -> ayahs.isEmpty

Reply:
Addressed.